Services/Concurrency/RateLimiting.cs

```diff
@@ -50,6 +86,16 @@ public class RateLimiting : IRateLimiting
             this.log = log;
             this.clusterSize = 1;
             this.instance = instance;
+
+            // Messaging quota variables
+            this.messagingQuotaLock = new object();
+            this.messagingQuotaExceeded = false;
+            this.messagingQuotaNextProbe = 0;
+
+            // Messaging quota variables
```

Comment:
Should it be Device quota variables?

Reply:
right, though all of this is going to change given the recent discussions